src/index.ts

```diff
@@ -59,7 +59,7 @@ export async function getPort (config?: GetPortInput): Promise<number> {
   return availablePort
 }
 
-async function checkPorts (ports: number[], host: string): Promise<number> {
+export async function checkPorts (ports: number[], host: string): Promise<number> {
```

Comment:
Since exporting, maybe set a default value for `host`? Would be nice to at least mention in `usage` section of readme